File: PracticaCode/PHP/division.php
<?php 
echo "Introduzca un número: ";
$numero1 = stream_get_line(STDIN, 1024, PHP_EOL);
echo "Introduzca un número: ";
$numero2 = stream_get_line(STDIN, 1024, PHP_EOL); 
echo "La division de " . $numero1 . "/" . $numero2 . " = " . ($numero1/$numero2);
?>

File: PracticaCode/C/division.c
#include <studio.h>
#include <studlib.h>
int main(){
	int numero1;
	int numero2;
	
	printf("Introduzca un número: ");
	scanf("%d", &numero1);
	printf("Introduzca un número: ");
	scanf("%d", &numero2);
	
	printf("A division de %d/%d=%d", numero1, numero2,(numero1/numero2));
	return 0;
	
}